week4-assignment/src/pages/SignupPage.tsx

```diff
@@ -0,0 +1,104 @@
+import { useState } from 'react';
+import SignupName from '../components/Signup/SignupName';
+import SignupPassword from '../components/Signup/SignupPassword';
+import SignupHobby from '../components/Signup/SignupHobby';
+import axios, { AxiosError } from 'axios';
+import { useNavigate } from 'react-router-dom';
+import { Container, Title, BottomTextWrapper, BottomButton } from './SignupPageStyle';
+
+const SignupPage = () => {
+  const [username, setUsername] = useState<string>(''); 
+  const [password, setPassword] = useState<string>(''); 
+  const [hobby, setHobby] = useState<string>(''); 
+  const [step, setStep] = useState<number>(1); 

```

Comment:
```suggestion
  const [username, setUsername] = useState(''); 
  const [password, setPassword] = useState(''); 
  const [hobby, setHobby] = useState(''); 
  const [step, setStep] = useState(1); 
```

이렇게 초기값을 지정해준 경우, type을 명시적으로 알려주지 않고, typescript가 자동으로 추론하게 하는 것이 좀 더 권장되는 방식이라고 합니다!

Reply:
오호.... 처음 알았습니다 !! 앞으로 이 점 생각해서 코드 작성하겠습니다! 감사합니다.